content/development/getting-started.html

```diff
@@ -17,7 +20,32 @@ <h3>Getting Involved</h3>
 and preferences are. Usually, by running Haiku for a while you will easily find
 out things that don't work as expected or that can be improved.
 
-<!--more-->
+<h2><a name="development_tools" title="development_tools"></a>What do I need for coding?</h2>
+<ul>
```

Comment:
what's with using lists everywhere again? This is clearly not a list

Reply:
It's a collection of different facts that have no continuity between each other but fall under a specific category.